terraform/application/config/qa.tfvars.json

```diff
@@ -1,5 +1,6 @@
 {
     "cluster": "test",
     "namespace": "bat-qa",
+    "deploy_azure_backing_services": true,
```

Comment:
this should be the default, so we don't normally specify it, same for enable_postgres_ssl below

Reply:
Removed